steps/pytorch/architectures/unet.py

```diff
@@ -1,13 +1,20 @@
 import torch
 import torch.nn as nn
+from .utils import get_downsample_pad, get_upsample_pad
 
 
 class UNet(nn.Module):
-    def __init__(self, conv_kernel,
-                 pool_kernel, pool_stride,
-                 repeat_blocks, n_filters,
-                 batch_norm, dropout,
-                 in_channels, **kwargs):
+    def __init__(self, conv_kernel=3,
+                 pool_kernel=3, pool_stride=2,
+                 repeat_blocks=2, n_filters=8,
+                 batch_norm=True, dropout=0.1,
+                 in_channels=3, out_channels=2,
+                 kernel_scale=3,
+                 **kwargs):
+
+        assert conv_kernel % 2 == 1
```

Comment:
@apyskir Can you add some User-facing information to both `assert`?

Reply:
Sure, I will add the info